README.md

```diff
@@ -1 +1,41 @@
-# hw01
\ No newline at end of file
+# hw01
+
+
+
+**Who am I?**
+========
+
+<img src="https://github.com/recalderamirez/hw01/blob/master/LUIS.jpg?raw=true" width="400">
+
+
+Hello!
+
+My name is _**Luis Recalde**_. I am from Paraguay. 
+
+I am a second MPP _(Master of Public Policy)_ student at the **Harris School of Public Policy.** My area of policy interest is International Development, broadly defined. During the Spring quarter I will be in Chicago, practicing self-quarantine (due to the COVID-19 pandemic) while "attending" and studying for my classes.
+
+### **My favorite classes** #
+
+In my two years studying a master's degree at **the University of Chicago** my favorite classes were:
+
+*   **Program Evaluation:** a class that teaches different econometrics methods used to evaluate the impact of programs. 
+
+*   **Cost-Benefit Analysis:** a class focused on tools to analyze the feasibility of projects through economic and statistical tools. 
+
+*   **Survey Research and Methodology:** a very practical course that teaches all steps concerning survey design and analysis.
+
+*   **Policy Lab:** a project-based course where I was part of a team that worked with the World Bank as a client on a project focused on survey-design.
+
+*   **Any other class?:** while this is only the first week of class, I have heard a lot of great things about _**Computing for the Social Sciences**_, and I am very much looking forward to including it into this list as well! 
+
+Thank you!    
+    
+        
+            
+_**Luis Recalde**_    
```

Comment:
I believe Luis did a good job adding links to several social networks, so the reader can explorer better who he is.